profiles/package.mask

```diff
@@ -36,6 +36,13 @@
 
 #--- END OF EXAMPLES ---
 
+# Jaco Kroon <jaco@uls.co.za> (2025-06-25)
+# Deprecate app-admin/fluentd, based on ruby.  Nightmare to maintain.
+# Please migrate to app-admin/fluent-bit.  If not possible, please take over
+# maintainership of this package.
+# Removal on 2025-07-25. Bug #833260
+app-admin/fluentd
```

Comment:
It'll be easier to implement this mask after the package is merged, I see your intention here though.

Reply:
This is a separate commit.  First commit is addition of new package.  Second commit is the masking.